src/main/scala/org/broadinstitute/dsde/workbench/sam/db/DbReference.scala

```diff
@@ -51,9 +50,9 @@ object DbReference extends LazyLogging {
     DbReference()
   }
 
-  def resource[F[_]: Sync](liquibaseConfig: LiquibaseConfig): Resource[F, DbReference] = Resource.make(
-    Sync[F].delay(init(liquibaseConfig))
-  )(_ => Sync[F].delay(DBs.closeAll()))
+  def resource(liquibaseConfig: LiquibaseConfig): Resource[IO, DbReference] = Resource.make(
+    IO(init(liquibaseConfig))
+  )(_ => IO(DBs.closeAll()))
```

Comment:
I got rid of all the Sync stuff because there was some problem with diverging implicit evidence and we only use IO anyway.